travis-scripts/script.sh

```diff
@@ -47,12 +48,14 @@ chmod -R go-w .
 if [ "$JOB_TYPE" = acceptance_tests_common ]
 then
     ./testall --tests=common
+    bash <(curl -s https://codecov.io/bash) -F acceptance_tests ;
```

Comment:
`acceptance_tests_common`

Reply:
They are called the same so it can merge coverage for distributed acceptance test executions to get same result as those whole suite was executed in single go.